src/qt/test/CMakeLists.txt

```diff
@@ -32,14 +32,6 @@ if(ENABLE_WALLET)
   )
 endif()
 
-if(NOT QT_IS_STATIC)
```

Comment:
In e0b79e029f15cf2e984c513e230b0d7c7914554d (build: Remove problematic code ): Can you add a sentence explaining why it's problematic?

Reply:
Please see https://github.com/bitcoin/bitcoin/pull/31147.